app/components/Analyst/ChangeStatus.tsx

```diff
@@ -206,15 +206,16 @@ const ChangeStatus: React.FC<Props> = ({
       !isAllowedExternalChange ||
       isInvalidConditionalApproval
     ) {
+      setDraftStatus(currentStatus);
```

Comment:
I think this is causing a small bug where the conditional approval details modal doesn't show up. To recreate it just use new form data or an application where the required conditional approval form fields `minister's decision` and `applicant response` aren't filled. 

It shoes this modal instead of the conditional approval modal
<img width="575" alt="Screenshot 2023-07-17 at 3 12 37 PM" src="https://github.com/bcgov/CONN-CCBC-portal/assets/14259474/3bcbaca3-68fc-4da1-98d4-e8faa09cbf2d">


conditional approval modal:
<img width="575" alt="Screenshot 2023-07-17 at 3 12 25 PM" src="https://github.com/bcgov/CONN-CCBC-portal/assets/14259474/11a04621-5cbf-4a61-b63e-7ce01be1a761">

Reply:
Thanks, looking into it

Reply:
Give it another try when you get a chance 😄

Reply:
Perfect 🙂